contrib/win32/openssh/install-sshd.ps1

```diff
@@ -84,6 +85,28 @@ if (Test-Path $moduliPath -PathType Leaf)
     Repair-ModuliFilePermission -FilePath $moduliPath @psBoundParameters -confirm:$false
 }
 
+#If %programData%/ssh folder already exists, verify permissions and fix, if necessary
+$sshProgDataPath = Join-Path $env:ProgramData "sshTest"
+if (Test-Path $sshProgDataPath)
+{
+    # Folder permission is FullAccess to System and Builtin/Admins and read only access to Authenticated users, if user allows
+    Repair-SshFolderFilePermission -FilePath $sshProgDataPath @psBoundParameters
+    # All files besides private key files and log folder/files should have same permissions as ssh folder, if user allows
+    $restricted_files = @("ssh_host_dsa_key", "ssh_host_ecdsa_key", "ssh_host_ed25519_key", "ssh_host_rsa_key", "*.logs")
```

Comment:
log files can have read only permissions for authenticated users

Reply:
Ok - I think the log files inherit from the log folder. 
When the log folder is created by SSHD, it only allows Full Access to System & Admins. 

I think either situation is permissible. Having a separate check for the log files/folders that permits read access for users, but won't add read access to users, if it has not already been granted might work best?